packages/env/lib/init-config.js

```diff
@@ -235,33 +236,32 @@ RUN apt-get -qy install $PHPIZE_DEPS && touch /usr/local/etc/php/php.ini
 # Set up sudo so they can have root access.
 RUN apt-get -qy install sudo
 RUN echo "$HOST_USERNAME ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers`;
-	} else {
-		dockerFileContent += `
+			if ( shouldEnableXdebug( config ) ) {
+				// Xdebug is only needed for WordPress environments.
+				dockerFileContent += getXdebugSetup( config.xdebug );
+			}
```

Comment:
How would you feel about merging `shouldEnableXdebug()` and `getXdebugSetup()` in favor of just having a `getXdebugConfiguration()` and inject that into `dockerFileContent` with `${ getXdebugConfiguration() }`? We can have the new function return an empty string if it's disabled and then return the configuration if it's enabled?

Reply:
yep, that sounds good!